bundles/CustomReportsBundle/config/pimcore/default.yaml

```diff
@@ -1,7 +1,6 @@
-pimcore:
-    newsletter:
+pimcore_newsletter:
```

Comment:
Yes, also the CMF would break because of this, but here at least we can add it as a dependency.
Maybe we should only document this for the custom reports that it is possible and remove it from the config so we do not have a dependency here?

Reply:
Decoupled from the custom reports bundle here https://github.com/pimcore/pimcore/pull/14848/commits/43ab13e3478de4dc7749213404c217e18c1e61c4

Reply:
It seems that there's a problem at the moment
![image](https://user-images.githubusercontent.com/6014195/231140313-ff44862d-7be5-451e-9176-6a0d6c1f8c56.png)